src/web/command_control/client/components/CommandControl.tsx

```diff
@@ -2247,6 +2247,8 @@ export default class CommandControl extends React.Component {
 	}
 
 	restoreUndo() {
+		if (!confirm('Click the OK button to go back to the previous task for all mission-active bots:')) return
```

Comment:
Instead of:
Click the OK button to go back to the previous task for all mission-active bots:
Use:
Click the OK button to undo the previous run edit that was made: